deploy/generate_addon.sh

```diff
@@ -41,14 +41,14 @@ rules:
     resources: [ 'loadbalancers' ]
     verbs: [ 'get', 'watch', 'list', 'update', 'create', 'delete' ]
   - apiGroups: [ 'kubevirt.io' ]
-    resources: [ 'virtualmachines' ]
+    resources: [ 'virtualmachines', 'virtualmachineinstances' ]
```

Comment:
in [here](https://github.com/harvester/harvester/blob/master/deploy/charts/harvester/templates/rbac.yaml#L129-L131) we have requested more verb permission, @yaocw2020 can u please help to confirm whether the `listWatch` permission is also needed?

Reply:
watch: https://github.com/harvester/cloud-provider-harvester/blob/ce82c01b7ef617b64481d6e4da0079799eeafb5a/pkg/cloud-controller-manager/loadBalancer.go#L97
no List usage